app/main/views/frameworks.py

```diff
@@ -725,39 +724,30 @@ def framework_updates_email_clarification_question(framework_slug):
     # Submit email to Zendesk so the question can be answered
```

Comment:
Can we change this comment so that we don't have an erroneous reference to Zendesk?

Reply:
```suggestion
    # Submit email to CCS so the question can be answered
```